src/components/FilterControls/MultiSelect/index.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useState, useEffect } from 'react'
+import OutlinedInput from '@mui/material/OutlinedInput'
+import InputLabel from '@mui/material/InputLabel'
+import MenuItem from '@mui/material/MenuItem'
+import FormControl from '@mui/material/FormControl'
+import Select, { SelectChangeEvent } from '@mui/material/Select'
+import Checkbox from '@mui/material/Checkbox'
+import ListItemText from '@mui/material/ListItemText'
+
+interface MultiSelectProps {
+  options: string[]
```

Comment:
I'd argue we don't need this to be a controlled component and can remove the options here